### PR TITLE
Soteria Branded First Aid Kit

### PR DESCRIPTION
**Pull request by AMcilraith**
## About The Pull Request
<details>
<summary>
	SI Branded First Aid Kits to compete with GP's Encroachement.
	See #5860 .
</summary>
<hr>

Adds a soteria branded first-aid kit to compete with the new greyson kits soon to be added.
Lowers price of soteria gauze and ointment in vendors, having the kit now similarly priced the intitial prices, but a bit more expensive. 
It's acutally cheaper to buy the kit considering the still very-high price of the gauze and ointment.
This is intended as SI's pre-emptive response to the greyson encroach on their medical monopoly.
	
<hr>
</details>

## Changelog
:cl:
add: soteria first aid kit filled with soteria gauze/ointment
add: soteria first aid kit to micromed priced at 750c
tweak: lowers prices of soteria gauze and ointment in micromed to 250c
/:cl:

### CODE REVIEW

**Comment by ChefDoggo** [comment]
aughgh... screenshot.... please......

**Comment by AMcilraith** [comment]
> aughgh... screenshot.... please......

It's literally one of our already existing sprites. Just the same medkit as the AFK, (red one) but purple. It's the same as all the other kits, uses an existing sprite

image

File: code/game/machinery/vendor/medical_vendors.dm [hunk 1]
```diff
@@ -72,12 +73,13 @@
...
 		/obj/item/stack/medical/bruise_pack = 50,
-		/obj/item/stack/medical/bruise_pack/soteria = 700,
+		/obj/item/stack/medical/bruise_pack/soteria = 250,
 		/obj/item/stack/medical/ointment/handmade  = 25,
 		/obj/item/stack/medical/ointment = 35,
-		/obj/item/stack/medical/ointment/soteria = 700,
+		/obj/item/stack/medical/ointment/soteria = 250,
 		/obj/item/stack/medical/splint/improvised = 5,
 		/obj/item/stack/medical/splint = 10,
+		/obj/item/storage/firstaid/regular/si = 750,
```
Comment (Trilbyspaceclone) [inline code/game/machinery/vendor/medical_vendors.dm:82]: This has about 1310cr value (1250 for bruise_pack and ointment + 50 + 10)
Maybe lower the price of the SI stuff a little more or remove the spare bandaids so its not as much of a swing in prices

Comment (AMcilraith) [reply code/game/machinery/vendor/medical_vendors.dm:82]: Alright. Will do.

**Pull request merged**